### ISSUE TITLE
Capturing "redo" command

### ISSUE THREAD

**Issue by porterchild** [ivangabriele/vscode-git-automator#234]
Not sure if I'm the only one who uses Ctrl + Shift + Z for 'redo', but the fact that this captures that combination, rendering the normal 'undo + redo' shortcut flow useless, seems a bit of an oversight.

**Comment by TheChiefMeat** [comment]
I just installed this addon, really helpful but the conflict with Ctrl + Shift + Z is a pain.

**Comment by paeolo** [comment]
Same here, really like the extension but "ctrl + shift + z" is commonly used for "redo". Is there a way we can change git automator keybindings ?
Anyway on my side I added this to my keybindings.json to keep the redo shortcut:
```json
  {
    "key": "ctrl+shift+z",
    "command": "redo"
  }
```

**Comment by jfgilmore** [comment]
Agreed, CMD+SHIFT+Z on a mac is the system default for undo. A better default for single file commits should be set.
